docs/usage.md

```diff
@@ -3,58 +3,166 @@
 
 # Usage
 
-To use this widget users needs to provide a JSON in a certain format for the widget to render data.
-This JSON data must be in a field of a record in a module.
-This widget can also be used to *Broadcast* an event, can be enabled in advanced settings in widget configuration. 
-
-*This widget can be used as representation of an individual record, if the filter criteria filters out multiple records, and the records contain data with the required format in the field then for each record a card will be formed.*
-
-
-If the broadcast event is turned on, the widget will then broadcast an event passing on the record IRI(Id) to the listning widgets.
-The widgets listning to the on-click event should be present on the dashboard as this widget.
-
-## Widget Preview and How to Use
-
-1. Where 'data' key is present inside the JSON provided
-
-    Edit View ![Edit View](./edit_view.png)
-
-    In the field, "Enter the key of object to be rendered" contains the main key of the object "recordSummaryCard", if there is no key for the object leave the field blank.
-
-``` JSON
-    "recordSummaryCard": {
+To use this widget users needs to provide a JSON in a certain format for the widget to render data. This JSON data must be in a field of a record in a module.
+
+This widget can also *Broadcast* an event. This option can be enabled in widget configuration's *Advanced Settings*. 
+
+This widget represents an individual record. If, however, multiple records meet the filter criteria, this widget creates a new card for each record.
+
+<table>
+    <td><strong>NOTE</strong></td>
+    <td>For the widget to create multiple cards, each record's JSON field must contain data in a very specific format.</td>
+</table>
+
+If the broadcast event is turned on, the widget will then broadcast an event passing on the record IRI(Id) to the listening widgets.
+The widgets listening to the on-click event should be present on the dashboard as this widget.
+
+<table>
+    <td><strong>IMPORTANT</strong></td>
+    <td>The <code>JSON</code> data to be rendered must have <strong><em>at least one</em></strong> of the following:
+        <ul>
+            <li><code>data</code> key</li>
+            <li><code>title</code> and <code>value</code> keys</li>
+        </ul>
+    </td>
+</table>
+
+Following are two acceptable JSON formats:
+
+<table>
+    <thead>
+        <th>With <code>data</code> Key</th>
+        <th>Without <code>data</code> Key</th>
+    </thead>
+    <tr>
+        <td>
+<pre>"recordTilesCustomerData": {

```

Comment:
You have used "summary-tile" data, use "summary-card" data, it is just below, there is no significance of 
the key-"value" in this widget

Use this as example
"recordSummaryCard": {
      "data": {
        "MTTD": "1 min",
        "MTTR": "2 mins",
        "Node Status": "Active",
        "Active Incidents": 0
      },
      "title": "Australia"
    }